src/main/java/org/jenkinsci/plugins/stashNotifier/BuildStatusUriFactory.java

```diff
@@ -8,9 +8,16 @@ public class BuildStatusUriFactory {
     private BuildStatusUriFactory() {
     }
 
-    public static URI create(String baseUri, String commit) {
```

Comment:
Can we leave this method around and create a new overload for the extra parameters?